ci-values.yaml

```diff
@@ -3,6 +3,11 @@ global:
   idamWebUrl: idam-web-public.aat.platform.hmcts.net
   devMode: true
 
+rpe-service-auth-provider:
+  java:
+    MICROSERVICEKEYS_CCD_ADMIN: AAAAAAAAAAAAAAAA
```

Comment:
do you also need the pay ones from chart-pay as well?

Reply:
I will create another PR for  payment integration. currently payment is disabled